test/tbot_web/controllers/tbot_controller_test.exs

```diff
@@ -0,0 +1,33 @@
+defmodule TbotWeb.PageControllerTest do
+  use TbotWeb.ConnCase
+
+  test "GET /bot with correct verify token returns challenge token and 200", %{conn: conn} do
+    test_conn = conn
+    |> put_req_header("accept", "application/json")
+    |> get("/bot", %{"hub.mode" => "subscribe", "hub.verify_token" => corrent_verify_token, "hub.challenge" => "blabla"})
```

Comment:
Shouldn't be `current_verify_token`?

Reply:
Ops, it is a typo. I actually meant to write `correct`.